Content.Server/Guardian/GuardianSystem.cs

```diff
@@ -188,6 +189,8 @@ private void UseCreator(EntityUid user, EntityUid target, EntityUid injector, Gu
             // Can only inject things with the component...
             if (!HasComp<CanHostGuardianComponent>(target))
             {
+                Loc.GetString("guardian-activator-invalid-target", ("entity", Identity.Name(target, EntityManager, user)));
```

Comment:
```suggestion
                var msg = Loc.GetString("guardian-activator-invalid-target", ("entity", Identity.Entity(target, EntityManager, user)));
```
- You'll need to pass in the entity instead of the name to make the Fluent functions work.
- You're not actually doing anything with the localized string here - save it to a variable so we can use it in the popup.